src/pages/DirectMessage.tsx

```diff
@@ -201,9 +251,40 @@ const DirectMessage = () => {
               <InfoOutlinedIcon />
             </IconButton>
           </Box>
-          <Divider />
+
           <Box sx={styles.chatContainer}>
             <List component="div" ref={messageRef} sx={styles.messageList}>
+              <Box
+                sx={styles.bioContainer}
+                onClick={() => navigate(`/${selectedConversation.username}`)}
+              >
+                {loading ? (
+                  <CircularProgress />
+                ) : (
```

Comment:
I'm not really against leaving it like this, but I think this conditional is sort of ugly here. Do you think it'd look nicer if we do 2 conditional renders rather than the ternary?

```
{loading && <CircularProgress />}
{!loading && (
  ... // Bio stuff
)}
```

Reply:
I like this suggestion, it helps improve the readability in the page.